src/indexer/utils/evm/index.ts

```diff
@@ -246,10 +247,20 @@ export async function saveDepositLogs(
   transferRepository: TransferRepository,
   depositRepository: DepositRepository,
   transferMap: Map<string, string>,
+  coinMarketCapService: CoinMarketCapService,
+  sharedConfig: SharedConfig,
 ): Promise<void> {
   let transfer = await transferRepository.findTransfer(decodedLog.depositNonce, Number(decodedLog.fromDomainId), Number(decodedLog.toDomainId))
   if (!transfer) {
-    transfer = await transferRepository.insertDepositTransfer(decodedLog)
+    const { amount, fromDomainId } = decodedLog
+
+    const currentDomain = sharedConfig.domains.find(domain => domain.id == parseInt(fromDomainId))
+
+    const tokenSymbol = currentDomain?.resources.find(resource => resource.resourceId == decodedLog.resourceID)?.symbol
+
+    const amountInUSD = await coinMarketCapService.getValueInUSD(amount, tokenSymbol!)
```

Comment:
What happens if API doesn't have price for specific symbol (for example bunch of testnet tokens). Can we handle this somehow, for example use 0 if API call fails (and log error). @wainola wdyt?